docs/security/configuring-pre-flight-checks.md

```diff
@@ -129,3 +129,17 @@ if is_promotion; then if user_is_allowed; then echo "Promotion allowed."; else f
 The list of users might come from other sources, for example [secrets](/essentials/using-secrets)
 or cloned repository. It is also possible to extend the functionality
 to restrict promotions from specific repositories branches. 
+
+### Dynamically editing your pipeline files
+
+During the initialization job, two important variables are exported:
+
+- `INPUT_FILE`: This variable points to the unmodified pipeline file, in this case, `semaphore.yml`.  
+- `OUTPUT_FILE`: This variable refers to a copy of the original pipeline file (`INPUT_FILE`).
+Ultimately, Semaphore uses this copied file to execute the pipeline.
```

Comment:
The output file is not just a simple copy of the input file, but actually, a compiled input file that can be used to fully define a pipeline. So, let's change these two lines to something like this:

- `OUTPUT_FILE`: This variable refers to a full pipeline file compiled from the original file (`INPUT_FILE`) during the [pipeline initialization](https://docs.semaphoreci.com/reference/pipeline-initialization/).
Ultimately, Semaphore uses this compiled file to define the pipeline that will be executed.